custom_functions/url_parse.json

```diff
@@ -16,34 +16,53 @@
     ],
     "outputs": [
         {
-            "contains_type": [],
+            "contains_type": [
+                "*"
+            ],
             "data_path": "scheme",
             "description": "The scheme of the URL, such as HTTP, HTTPS, or FTP."
         },
         {
-            "contains_type": [],
+            "contains_type": [
+                "domain"
+            ],
             "data_path": "netloc",
             "description": "The network location of the URL, which is typically the hostname."
         },
         {
-            "contains_type": [],
+            "contains_type": [
+                "*"
+            ],
             "data_path": "path",
             "description": "The path to the resource after the first slash in the URL, such as \"en_us/software/splunk-security-orchestration-and-automation.html\"."
         },
         {
-            "contains_type": [],
+            "contains_type": [
+                "*"
+            ],
             "data_path": "params",
             "description": "The parameters in the URL after the semicolon."
         },
         {
-            "contains_type": [],
+            "contains_type": [
+                "*"
+            ],
             "data_path": "query",
             "description": "The query string of the URL after the question mark. Multiple parameters are not separated from each other."
         },
         {
-            "contains_type": [],
+            "contains_type": [
+                "*"
+            ],
             "data_path": "fragment",
             "description": "The subcomponent of the resource which is identified after the hash sign."
+        },
+        {
+            "contains_type": [
+                "url"
+            ],
+            "data_path": "originalUrl",
```

Comment:
Could you please use the variable name "input_url" so it is consistent with the lower case and underscore we use elsewhere, and also with the naming we use in other similar outputs like in the passthrough custom function?

Reply:
fixed in https://github.com/phantomcyber/playbooks/pull/17/commits/0bf33e6b8c69ee5827884235295276ec96ae12c7